src/js/components/put-away/PutAwaySecondPage.jsx

```diff
@@ -625,6 +638,8 @@ class PutAwaySecondPage extends Component {
         }
         <div className="submit-buttons">
           <button
+            disabled={_.some(this.state.putAway.putawayItems, putawayItem =>
+                (putawayItem.quantity > putawayItem.quantityAvailable) || putawayItem.quantity < 1)}
```

Comment:
I can see a similar logic here, but I assume it is not inside the `Cell`, so how about creating a function and pass there two arguments - the function that could be used for this case and above cases?